src/components/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Header = () => {
+
```

Comment:
[formatting] :scissors: the extra empty line. Consider converting to implicit return as well.